.github/workflows/lint.yml

```diff
@@ -75,10 +75,10 @@ jobs:
     steps:
```

Comment:
Nit: I think there are a few more references to Py11 in this file.

Reply:
I'll update lint workflow in a followup pr if it's fine for you

Reply:
Fixed